crates/bvs-guardrail/src/msg.rs

```diff
@@ -0,0 +1,112 @@
+use bvs_vault_router::state::SlashingRequestId;
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Addr, CosmosMsg, Empty};
+use cw3::{DepositInfo, Status, Vote};
+use cw_utils::{Expiration, Threshold, ThresholdResponse};
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub owner: String,
+    pub voters: Vec<Voter>,
+    pub threshold: Threshold,
+}
+
+#[cw_serde]
+pub enum ExecuteMsg {
+    Propose {
+        slash_request_id: SlashingRequestId,
```

Comment:
```suggestion
        slashing_request_id: SlashingRequestId,
```
Should we standardize this? Or keep it as `proposal_id`?